cogs/direct_message.py

```diff
@@ -99,6 +99,14 @@ async def send_mail(self, message, guild):
                         topic=f"ModMail Channel {message.author.id} {message.channel.id} (Please "
                         "do not change this)",
                     )
+                elif "Maximum number of channels in category reached" in e.text:
+                    await message.channel.send(
+                        ErrorEmbed(
```

Comment:
I would add a title to the Embed, such as "Too Many Channels"

I would also shorten the filter phrase to "Maximum number of channels" as that is specific enough for Discord and anything longer is more likely to get broken if Discord changes message.

Reply:
The embed title is redundant information, the content already says that the server has hit the cap